.github/actions/report-run-failure/README.md

```diff
@@ -22,7 +30,7 @@ jobs:
 
       # copy lines below to report a github failure to your team channel
       - if: ${{ failure() }}
-        uses: alphagov/govuk-infrastrtucture/.github/actions/report-run-failure@main
+        uses: alphagov/govuk-infrastructure/.github/actions/report-run-failure.yml@main
```

Comment:
Does this need `.yml` to work?

Also does the `action.yaml` file need renaming to `action.yml` for this action

Reply:
It doesn't need: _"Workflow files use YAML syntax, and must have either a .yml or .yaml file extension."_
I renamed it for consistency (as mentioned in the commit message) as all 10 other actions use `.yml` extension.